modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -149,19 +149,7 @@ const Body = function (bodyValue) {
 };
 
 exports.WebRequest = function (method, path) {
-  const tildeHeaders = {
-    '~server_protocol': function () {
-      return dRequest.getProtocol()
-    },
-    '~server_name': function () {
-      return dRequest.getServerName()
-    },
-    '~server_port': function () {
-      return dRequest.getServerPort()
-    },
-    '~request_line': function (webRequest) {
-      return WEB_UTILS.getMethodName(webRequest.method) + ' ' + webRequest.path + ' ' + dRequest.getProtocol()
-    },
+  var tildeHeaders = {
```

Comment:
Use `const`

Reply:
It was being reassigned, we agreed that all of the ~ headers should be removed anyway